puppet/zulip/files/postgresql/process_fts_updates

```diff
@@ -27,7 +30,21 @@ logger.setLevel(logging.DEBUG)
 
 logger.info("process_fts_updates starting")
 
-conn = psycopg2.connect("user=zulip")
+sys.path.insert(0, '/home/zulip/deployments/current')
+os.environ['DJANGO_SETTINGS_MODULE'] = 'zproject.settings'
+import zproject.settings
```

Comment:
The way `from django.conf import settings` works is that it ends up importing `zproject.settings`

So if you write this sequence:
```
os.environ['DJANGO_SETTINGS_MODULE'] = 'zproject.settings'
from django.conf import settings
```
then you can access the settings via just `settings.REMOTE_POSTGRES_SSLMODE` or `settings.get_secret`

See for example `tools/minify-js`.